tutorials/turbo_1/turbo_1.ipynb

```diff
@@ -402,135 +399,98 @@
+      "228) Best value: -3.63e+00, TR length: 1.00e-01\n",
+      "232) Best value: -3.08e+00, TR length: 1.00e-01\n",
+      "236) Best value: -2.89e+00, TR length: 1.00e-01\n",
+      "240) Best value: -2.89e+00, TR length: 1.00e-01\n",
+      "244) Best value: -2.89e+00, TR length: 1.00e-01\n",
+      "248) Best value: -2.89e+00, TR length: 1.00e-01\n",
+      "252) Best value: -2.89e+00, TR length: 1.00e-01\n",
+      "256) Best value: -2.89e+00, TR length: 5.00e-02\n",
+      "260) Best value: -2.89e+00, TR length: 5.00e-02\n",
+      "264) Best value: -2.67e+00, TR length: 5.00e-02\n",
+      "268) Best value: -2.53e+00, TR length: 5.00e-02\n",
+      "272) Best value: -2.52e+00, TR length: 5.00e-02\n",
+      "276) Best value: -2.52e+00, TR length: 5.00e-02\n",
+      "280) Best value: -2.52e+00, TR length: 5.00e-02\n",
+      "284) Best value: -2.35e+00, TR length: 5.00e-02\n",
+      "288) Best value: -2.24e+00, TR length: 5.00e-02\n",
+      "292) Best value: -2.24e+00, TR length: 5.00e-02\n",
+      "296) Best value: -2.20e+00, TR length: 5.00e-02\n",
+      "300) Best value: -2.20e+00, TR length: 5.00e-02\n",
+      "304) Best value: -2.20e+00, TR length: 5.00e-02\n",
+      "308) Best value: -2.20e+00, TR length: 5.00e-02\n",
+      "312) Best value: -2.16e+00, TR length: 5.00e-02\n",
+      "316) Best value: -1.73e+00, TR length: 5.00e-02\n",
+      "320) Best value: -1.73e+00, TR length: 5.00e-02\n",
+      "324) Best value: -1.30e+00, TR length: 5.00e-02\n",
+      "328) Best value: -1.30e+00, TR length: 5.00e-02\n",
+      "332) Best value: -1.30e+00, TR length: 5.00e-02\n",
+      "336) Best value: -1.30e+00, TR length: 5.00e-02\n",
+      "340) Best value: -1.30e+00, TR length: 5.00e-02\n",
+      "344) Best value: -1.30e+00, TR length: 2.50e-02\n",
+      "348) Best value: -1.22e+00, TR length: 2.50e-02\n",
+      "352) Best value: -1.22e+00, TR length: 2.50e-02\n",
+      "356) Best value: -1.22e+00, TR length: 2.50e-02\n",
+      "360) Best value: -1.22e+00, TR length: 2.50e-02\n",
+      "364) Best value: -1.22e+00, TR length: 2.50e-02\n",
+      "368) Best value: -1.22e+00, TR length: 1.25e-02\n",
+      "372) Best value: -1.17e+00, TR length: 1.25e-02\n",
+      "376) Best value: -1.15e+00, TR length: 1.25e-02\n",
+      "380) Best value: -1.15e+00, TR length: 1.25e-02\n",
+      "384) Best value: -1.15e+00, TR length: 1.25e-02\n",
+      "388) Best value: -1.15e+00, TR length: 1.25e-02\n",
+      "392) Best value: -1.15e+00, TR length: 1.25e-02\n",
+      "396) Best value: -1.15e+00, TR length: 6.25e-03\n"
      ]
     }
    ],
    "source": [
     "X_turbo = get_initial_points(dim, n_init)\n",
-    "Y_turbo = torch.tensor(\n",
-    "    [eval_objective(x) for x in X_turbo], dtype=dtype, device=device\n",
-    ").unsqueeze(-1)\n",
+    "Y_turbo = torch.tensor([eval_objective(x) for x in X_turbo], dtype=dtype, device=device).unsqueeze(\n",
+    "    -1\n",
+    ")\n",
```

Comment:
This gets rather long and exceeds the cell width on some layouts; mind switching this back?

Reply:
Will do, this was auto formatting locally with ruff so I will look into the repo settings and see if there's anything to set such that it auto reverts.